Docs/sphinx/InputFiles.rst

```diff
@@ -226,6 +226,22 @@ Additionally, tagging is supported for a user-specified species which can functi
    tagging.ftracerr = 150.e-6
 
 Users can specify their own tagging criteria in the `prob.H` of their case. An example of this is provided in the Taylor-Green regression test.
+
+In addition to these tagging criteria, one can use tagging functionality provided by AMReX. Here are examples of how that is done:
+
+::
+
+   # Tag inside a box and a velocity magnitude value
+   tagging.refinement_indicators = yLow magvel
+   tagging.yLow.in_box_lo = -0.1  -0.52  -0.85
+   tagging.yLow.in_box_hi =  3.1 -0.45    0.85
+
+   tagging.magvel.max_level     = 2
+   tagging.magvel.value_greater = 1.2e4
+   tagging.magvel.field_name    = magvel
+
+The following keys are implemented: `value_greater`, `value_less`, `vorticity_greater`, `adjacent_difference_greater`, `in_box_lo` and `in_box_hi` (to specify a refinement region), `max_level`, `start_time`, and `end_time`.
```

Comment:
Can you add something like `field_name` may be be any derived or state variable?

Reply:
Good idea. Done.